docs/index.rst

```diff
@@ -22,7 +22,7 @@ We have in-person and virtual conferences around the world:
 
    - :doc:`Australia 2023 </conf/australia/2023/index>`, **December 7-8, 2023**, Melbourne, Australia
```

Comment:
Not 100% sure how this got mixed up, but in `main` this should have been updated to the 2024 date: https://github.com/writethedocs/www/blob/main/docs/index.rst

Reply:
That'll be because Swapnil merged the post conf post. The merge resolution should take care of it.